relayer/pkg/chainlink/ocr2/medianreport/report.go

```diff
@@ -52,6 +53,9 @@ func (c ReportCodec) BuildReport(oo []median.ParsedAttributedObservation) (types
 	juelsPerFeeCoin = starknet.SignedBigToFelt(juelsPerFeeCoin) // converts negative bigInts to corresponding felt in bigInt form
```

Comment:
Do we expect the `juelsPerFeeCoin` to ever be a negative number? 
We should probably error here if that happens, right? And probably the same for the `gasPrice`.

Reply:
We don't expect it to be negative, but the EVM codec omits this check too: https://github.com/smartcontractkit/libocr/blob/cdfa6bef133ac1c69042ff04295ab65d867837c9/offchainreporting2/reportingplugin/median/evmreportcodec/reportcodec.go#L49-L53

I think it's done somewhere inside libocr maybe?